### PR TITLE
DAPP

### PR DESCRIPTION
**Pull request by josephguillaume**
Initial version by @Caroline-nc , edited by Joseph

### CODE REVIEW

File: _theory/adaptation-pathways.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: under-construction
+title: Adaptation pathways
+permalink: /theory/adaptation-pathways/
+excerpt: TODO
```
Comment (josephguillaume) [inline _theory/adaptation-pathways.md:5]: @Caroline-nc - Could you please add a short description of adaptation pathways that contrasts them with adaptive pathways?

File: _theory/adaptive-policy-making.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+layout: under-construction
+title: Adaptive policy making
+permalink: /theory/adaptive-policy-making/
+excerpt: TODO
```
Comment (josephguillaume) [inline _theory/adaptive-policy-making.md:5]: @Caroline-nc : Could you please add a short description of adaptive policy making?

File: _theory/pathway-dependency.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+layout: under-construction
+title: Path dependency
+permalink: /theory/path-dependency/
+excerpt: TODO
```
Comment (josephguillaume) [inline _theory/pathway-dependency.md:5]: @Caroline-nc : Could you please add a short definition of path dependency, preferrably as used by DAPP?

File: _theory/tipping-points.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+layout: under-construction
+title: Tipping points
+permalink: /theory/tipping-points/
+excerpt:
```
Comment (josephguillaume) [inline _theory/tipping-points.md:5]: @Caroline-nc - could you please add a general definition of tipping points. We probably want to cover both the DAPP usage (i.e. failure) and the broader resilience idea.

File: _theory/DAPP.md [hunk 5]
```diff
@@ -1,26 +1,122 @@
...
+  </ul></p>
+requirements: >
+  <p> Operationalising the DAPP approach requires: <ul>
+  <li> setting the scene for a system of interest and defining a vision [2] </li>
+  <li> defining narratives describing future plausible conditions [2] </li>
+  <li> identifying indicators of change for the system and for actions and associated threshold values [2]</li>
+  <li> stakeholders' buy-in and engagement in relevant activities [3] </li>
+  <li> considering the intervention of local knowledge-brokers to facilitate the application of the approach and stakeholder engagement [3] </li>
+  </ul></p>
+decomposition: >
```
Comment (josephguillaume) [inline _theory/DAPP.md:26]: @Caroline-nc - Would it be possible to add a figure? We can discuss.

File: _theory/DAPP.md [hunk 6]
```diff
@@ -1,26 +1,122 @@
...
+  <p> The DAPP framework can be decomposed into eight main iterative steps [2]:<ol>
+  <li> Participatory problem framing, defining:
+   <ul>
+   <li>boundaries of a system of interest</li>
+   <li>performance objectives, including conditions of success for the system used to evaluate performance of policy options</li>
+   <li>current and future vulnerabilities</li>
+   </ul> 
+  Identified vulnerabilities are used to :<ul>
+   <li> select plausible policy measures (based on manageable vulnerabilities)</li>
+   <li> define plausible narratives about future changes (based on non-manageable vulnerabilities) that could be either static (i.e., with an end point into the future) or transient (describing developments over time) [10]</li></ul></li>
```
Comment (josephguillaume) [inline _theory/DAPP.md:36]: It's not yet clear how vulnerabilities are used for this. Is it possible to add a little more detail?
Is it also possible to integrate this into the list above? Or should it be a separate step?

**Comment by Caroline-nc** [comment]
I was trying to figure out how to add figures last time, it would help a lot. I understand that you may need a code to insert pictures?

File: _theory/DAPP.md [hunk 7]
```diff
@@ -1,26 +1,122 @@
...
+  <li> when applied to facilitate decision-making understanding and to support the acknowledgement of uncertainties in plans </li>
+  <li> as an enabler for action </li>
+  <li> when considering coupled physical-technical-human systems and long time planning horizons </li>
+  <li> when considering the potential for implementing alternative adaptation actions to solve issues according to how the future may unfold </li> 
+  </ul></p>
+requirements: >
+  <p> Operationalising the DAPP approach requires: <ul>
+  <li> setting the scene for a system of interest and defining a vision [2] </li>
+  <li> defining narratives describing future plausible conditions [2] </li>
+  <li> identifying indicators of change for the system and for actions and associated threshold values [2]</li>
```
Comment (josephguillaume) [inline _theory/DAPP.md:22]: Are these all covered in step 1 below, and should be deleted here?

File: _theory/DAPP.md [hunk 8]
```diff
@@ -1,26 +1,122 @@
...
+  <li> stakeholders' buy-in and engagement in relevant activities [3] </li>
+  <li> considering the intervention of local knowledge-brokers to facilitate the application of the approach and stakeholder engagement [3] </li>
```
Comment (josephguillaume) [inline _theory/DAPP.md:24]: If possible, would be good to be more specific about how knowledge-brokers would help facilitate?

File: _theory/DAPP.md [hunk 9]
```diff
@@ -1,26 +1,122 @@
...
 title: Dynamic Adaptive Policy Pathways (DAPP)
 permalink: /theory/DAPP/
-excerpt:
-purpose: Purpose
-requirements: Requirements
-decomposition: 3 steps
+excerpt: DAPP is a framework to evaluate policy robustness and adaptiveness under deep uncertainty. The approach combines adaptive policymaking and adaptive pathways concepts.
+purpose: >
+  <p>DAPP  [1] aims at guiding the design of a robust and adaptive plan as a sequence of actions (near term and alternative future actions) over time through [2]: <ul>
+  <li>exploration of sequences of decisions (<a href="../adaptation-pathways/">adaptation pathways</a>) under plausible futures (i.e., scenarios)</li>
```
Comment (josephguillaume) [inline _theory/DAPP.md:8]: Was this meant to be "adaptive pathways"?

File: _theory/DAPP.md [hunk 10]
```diff
@@ -1,26 +1,122 @@
...
+  <li> making visible <a href="../path-dependency/">path dependency</a> of alternative future strategies</li></ul></p>
+  <p> DAPP also supports participatory modelling through engaging stakeholders and decision makers in problem-solving activities [2,3]</p>
+  <p> The approach was originally built to implement climate-resilient pathways for long-term water resources strategic planning but can be applied to other long-term strategic planning problems under <a href="../DMDU/">deep uncertainty</a> [2,13] </p> 
+  <p> The approach is useful [2]: <ul>
+  <li> when applied to facilitate decision-making understanding and to support the acknowledgement of uncertainties in plans </li>
+  <li> as an enabler for action </li>
```
Comment (josephguillaume) [inline _theory/DAPP.md:14]: These are a bit vague. Reword or delete?

File: _theory/DAPP.md [hunk 11]
```diff
@@ -1,26 +1,122 @@
...
+   <li> Identification of adaptation <a href="../tipping-points/">tipping points</a> (ATPs), i.e. conditions and timing of failure for the current system when facing different future scenarios. Approaches to identify ATPs (i.e., threshold values) include: <ul>
+   <li> 'bottom-up' approaches: <a href="../stress-testing/">stress tests</a> through sensitivity analyses, <a href="../scenario-discovery/">scenario discovery</a> [5], expert judgement [1], and/or stakeholder consultation [3]</li>
+   <li> 'top-down' approaches: model-based assessment using static or transient [9] <a href="../scenario-development/">scenarios</a></li></ul>
+  </li>
+  <li> Identification of alternative actions based on step 1 and 2, to address identified vulnerabilities or opportunities, followed by determination of their ATPs </li>
+  <li> Design and assessment of the <a href="../adaptive-pathways">adaptive pathways</a>. This stage leads to the design of a pathway map, also defined as "metro map", to achieve a performance objective for the system.
```
Comment (josephguillaume) [inline _theory/DAPP.md:42]: Can we provide an example metro map here?

File: _theory/DAPP.md [hunk 12]
```diff
@@ -1,26 +1,122 @@
...
+  </li>
+  <li> Identification of alternative actions based on step 1 and 2, to address identified vulnerabilities or opportunities, followed by determination of their ATPs </li>
+  <li> Design and assessment of the <a href="../adaptive-pathways">adaptive pathways</a>. This stage leads to the design of a pathway map, also defined as "metro map", to achieve a performance objective for the system. 
+  <p>Opportunities, no-regret actions, lock-ins and their associated timing of implementation under changing conditions can be inferred from the map.</p>
+  Methods to develop and explore adaptive pathways include:<ul>
+   <li>drawing them manually</li>
+   <li>using a dedicated application such as the <a href="../../tools/pathways-generator/">Pathways Generator</a> [4] </li>
+   <li> based on models such as agent-based models [14], using multi-objective robust optimisation [6], serious games [7,8] </li>
+   <li> during stakeholder focus group discussions [3]</li>
+   <li> using a combimation of model-based assessment and group discussions </li></ul>
```
Comment (josephguillaume) [inline _theory/DAPP.md:49]:
```suggestion
   <li> using a combination of model-based assessment and group discussions </li></ul>
```

**Comment by josephguillaume** [comment]
> I was trying to figure out how to add figures last time, it would help a lot. I understand that you may need a code to insert pictures?

The image is added to the images folder (with a sufficiently unique name), and then the img html tag is used. A text description is also provided with the "alt" tag. Here's an example: 
```HTML
<img src="../../images/maier-2016-fig1.jpg" width=50% alt="The difference between describing future system states through: a) anticipating the future based on best available knowledge, b) quantifying future uncertainty, c) exploring multiple plausible futures, d) combining the three paradigms to address different sources of uncertainty within a problem".>
```

**Comment by Caroline-nc** [comment]
Thanks Joseph,

I am going to have a try tomorrow to try to insert the Dapp framework.

Have a lovely evening,

Cheers,
Caroline

File: _theory/DAPP.md (hunk 9 above)
Comment (josephguillaume) [reply _theory/DAPP.md]: Thanks. I've moved the discussion of adaptive vs adaptive pathways to their respective pages

File: _theory/DAPP.md [hunk 13]
```diff
@@ -1,26 +1,131 @@
...
 title: Dynamic Adaptive Policy Pathways (DAPP)
 permalink: /theory/DAPP/
-excerpt:
-purpose: Purpose
-requirements: Requirements
-decomposition: 3 steps
+excerpt: DAPP is a framework to evaluate policy robustness and adaptiveness under deep uncertainty. The approach combines adaptive policymaking and adaptive pathways concepts.
+purpose: >
+  <p>DAPP  [1] aims at guiding the design of a robust and adaptive plan as a sequence of actions (near term and alternative future actions) over time through [2]: <ul>
+  <li>exploration of sequences of decisions through an <a href="../adaptation-pathways/">adaptation pathways</a> approach to provide an <a href="../adaptive-pathways/">adaptive pathway</a> under plausible futures (i.e., scenarios)</li>
```
Comment (josephguillaume) [inline _theory/DAPP.md:8]: @Caroline-nc - are you happy with this wording?
It avoids explicitly discussing the difference between the two on this page but hints at the distinction

File: _theory/DAPP.md (hunk 11 above)
Comment (josephguillaume) [reply _theory/DAPP.md]: Thanks!

**Comment by Caroline-nc** [comment]
Hello Joseph,

I am not too sure about the sentence, it could be confusing; the last sentence making sound as if you are selecting one route over all plausible ones.
Maybe try:

+  <li>exploration of sequences of decisions through an <a href="../adaptation-pathways/">adaptation pathways</a> approach to provide a portfolio of plausible robust and adaptive actions, made of near term and long term options,(i.e.,<a href=”../adaptive-pathways/”>adaptive pathways</a>) for a system to cope under plausible futures (i.e., scenarios)</li>

File: _theory/tipping-points.md (hunk 4 above)
Comment (josephguillaume) [reply _theory/tipping-points.md:5]: Thanks. Could you please edit the tipping-points.md file, adding the excerpt in particular.

File: _theory/pathway-dependency.md (hunk 3 above)
Comment (josephguillaume) [reply _theory/pathway-dependency.md:5]: Thanks. Could you please edit the pathway-dependency.md file

File: _theory/adaptive-pathways.md [hunk 14]
```diff
@@ -2,8 +2,9 @@
 layout: under-construction
 title: Adaptive pathways
 permalink: /theory/adaptive-pathways/
-excerpt: Adaptive pathways describe the sequences of options that can be taken and the circumstances in which alternative sequences might be selected. Planning full pathways enables investment in an initial management option knowing that changes can be made during implementation of the plan.
-purpose: Purpose
+excerpt: Adaptive pathways consist of sequences of options that can be taken and the circumstances in which alternative sequences might be selected. Adaptiveness is an attribute of a system allowing it to cope with different future uncertainties. Planning of "adaptation pathways" contributes to adaptiveness by enabling investment in an initial management option knowing that changes can be made during implementation of the plan.
```
Comment (josephguillaume) [inline _theory/adaptive-pathways.md:5]: @Caroline-nc - could you please check this excerpt and suggest edits if needed?

File: _theory/adaptation-pathways.md [hunk 15]
```diff
@@ -0,0 +1,11 @@
+---
+layout: under-construction
+title: Adaptation pathways
+permalink: /theory/adaptation-pathways/
+excerpt: Adaptation pathways is an analytical approach for undertaking planning and implementation under uncertainty and change. It provides a process for deciding when and under which conditions acceptable actions need to be implemented to cope with multiple plausible futures. In contrast, an "adaptive" pathway refers to the ability of a system to cope with different future uncertainties.
```
Comment (josephguillaume) [inline _theory/adaptation-pathways.md:5]: @Caroline-nc - this is based on your text. Could you please check and edit if needed?

File: _theory/DAPP.md [hunk 16]
```diff
@@ -1,26 +1,139 @@
...
 also_see:
-  - permalink: /tools/tool-1/
-    name: tool 1
-    description: Description of tool 1
+  - permalink: /theory/scenario-discovery/
+    name: Scenario Discovery
+    description: A form of vulnerability analysis that can be used within DAPP (Step 2) to identify policy-relevant plausible future scenarios or a limited of future scenarios to focus on.
+  - permalink: /theory/adaptive-policy-making/
+    name: Adaptive policy making
+    description: TODO
```
Comment (josephguillaume) [inline _theory/DAPP.md:90]: @Caroline-nc - could you please add a short description, describing how adaptive policy making relates to DAPP?

File: _theory/DAPP.md [hunk 17]
```diff
@@ -1,26 +1,139 @@
...
+  - permalink: /theory/adaptive-pathways/
+    name: Adaptive pathways
+    description: Descriptions of the sequences of options that can be taken and the circumstances in which alternative sequences might be selected.
+  - permalink: /theory/adaptation-pathways/
+    name: Adaptation pathways
+    description: TODO
```
Comment (josephguillaume) [inline _theory/DAPP.md:96]: @Caroline-nc - could you please add a short description, specifically contrasting with the description of adaptive pathways above it?

File: _theory/adaptive-policy-making.md (hunk 2 above)
Comment (josephguillaume) [reply _theory/adaptive-policy-making.md:5]: Thanks. Could you please add the excerpt. Other text can be added to the file now already if you want - it won't show up on the live site anyway.

**Comment by josephguillaume** [comment]
@Caroline-nc - thanks for all your work. Could you please have a look at the remaining issues/comments?
Could you also check whether you're happy with the revised version/my edits?

**Comment by Caroline-nc** [comment]
No problem, Joseph.

Will update comments by tomorrow.

Cheers,
Caroline

File: _theory/adaptive-policy-making.md (hunk 2 above)
Comment (Caroline-nc) [reply _theory/adaptive-policy-making.md:5]: Good morning Joseph,
I have committed under a new branch, DAP, the details for the approach. DAP another way of naming (Dynamic) Adaptive Planning. There are not many examples of applications in the water field except one used to illustrate the approach and appears to be a theoretical example. DAP has been mostly used in the transportation sector. 
As for tools, I could not find any clear mention of tools. I know that EMA could be used for evaluating vulnerabilities, but there is no evidence that the EMAworkbench has been used with DAP.
I will start later today with the article for "adaptive pathways".

Cheers,
Caroline

**Pull request merged**